name-registry/tests/utils/abi.rs

```diff
@@ -0,0 +1,90 @@
+use crate::utils::*;
+
+pub async fn extend(instance: &MyContract, name: &String, duration: u64) {
```

Comment:
These tests should be returning a callresponse so that logs can be checked. I think that is now testable in the latest SDK which the repo has been bumped to